extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/ResteasyReactiveRuntimeRecorder.java

```diff
@@ -38,8 +38,8 @@ public void configure(RuntimeValue<Deployment> deployment,
         List<RuntimeConfigurableServerRestHandler> runtimeConfigurableServerRestHandlers = deployment.getValue()
                 .getRuntimeConfigurableServerRestHandlers();
         deployment.getValue().setRuntimeConfiguration(runtimeConfiguration);
-        for (RuntimeConfigurableServerRestHandler handler : runtimeConfigurableServerRestHandlers) {
-            handler.configure(runtimeConfiguration);
+        for (int i = 0; i < runtimeConfigurableServerRestHandlers.size(); i++) {
+            runtimeConfigurableServerRestHandlers.get(i).configure(runtimeConfiguration);
```

Comment:
Is there a reason why you are doing that? It doesn't look like a hotspot?

Reply:
It's not a hotspot, but we try to limit allocations in RR wherever possible